### PR TITLE
Added more domains from mytemp.email

### PR DESCRIPTION
**Pull request by deguif**
See https://mytemp.email/

### CODE REVIEW

File: disposable_email_blacklist.conf [hunk 1]
```diff
@@ -1544,6 +1560,7 @@ secretemail.de
 secure-mail.biz
 secure-mail.cc
 secured-link.net
+securehost.com.es
```
Comment (martenson) [inline disposable_email_blacklist.conf]: This one does not seem as a disposable address host (http://securehost.com.es/)
Where are you getting these?

Comment (deguif) [inline disposable_email_blacklist.conf]: It was proposed by https://mytemp.email/ when you ask to create a new disposable email address

Comment (deguif) [inline disposable_email_blacklist.conf]: You're right, the host doesn't seem to be a disposable domain host. Maybe the domain was sold to a new owner and they use it now for hosting their website. I'm going to remove this one.

Comment (martenson) [inline disposable_email_blacklist.conf]: I see. Could you please drop this one out?

**Comment by martenson** [comment]
Also I am not sure about this, but generally we probably want to avoid putting domains of the third level here. If spammer/disposable service uses somedomain.com they can also easily create <anything_here>.somedomain.com so we should filter on the top domain name.

File: disposable_email_blacklist.conf (hunk 1 above)
Comment (frin) [inline disposable_email_blacklist.conf]: Interesting, the securehost.com.es website doesn't work now (Connection refused).

Comment (deguif) [inline disposable_email_blacklist.conf]: But http://www.wifiber.es/ which was the same site.

**Comment by deguif** [comment]
Sounds legit.
In this case we should update the script to check email validity to check only for second domain level and in some cases for third domain level (for domains like *.co.uk or *.com.es, etc...)

**Comment by deguif** [comment]
Here the pseudo PHP code that should handle the new behavior.

```php
<?php
//check-disposable-email.php

if (!isset($argv[1])) {
   echo "You should specify a domain to check\n";
   exit(1);
}

$domain = $argv[1];
$disposableDomains = [
    'disposable.co.uk',
    'disposable.com',
];

if (preg_match('/(?:[^.]+\.)?([^.]*\.[^.]+)$/', $domain, $matches)) {
   $twoLevelDomain = $matches[1];
   $threeLevelDomain = $matches[0] !== $matches[1] ? $matches[0] : null;

   echo 'Two level domain: '.$twoLevelDomain."\n";
   if (null !== $threeLevelDomain) {
       echo 'Three level domain: '.$threeLevelDomain."\n";
   }

   foreach ($disposableDomains as $disposableDomain) {
       if ($disposableDomain === $twoLevelDomain || ($threeLevelDomain && $disposableDomain === $threeLevelDomain)) {
           echo sprintf("Domain %s is a disposable domain.\n", $domain);
           return true;
       }
   }

   echo sprintf("Domain %s is not a disposable domain.\n", $domain);
   return false;
}
```

**Comment by martenson** [comment]
@deguif I would like to leave the implementation details on the deployer to keep this list simple and not opinionated. We can provide hints and pointers though.

**Pull request merged**